data/boarding missions.txt

```diff
@@ -37,3 +37,29 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+mission "Navy emergency"
+	assisting
+	repeat
+	deadline
+	passengers 1 3
+	description "Deliver <fare> to <destination> by <date>; the military hospital is waiting for you."
+	to offer
+		"reputation: Navy (Oathkeeper)" > 100
```

Comment:
Why only if the player's reputation is so high?

Reply:
Trust

Reply:
They need to trust him